README.md

```diff
@@ -120,23 +120,23 @@ to install/update.
     ```python
     import tikzplotlib
     tikzplotlib.save("mytikz.tex")
+    # or
+    tikzplotlib.save("mytikz.tex", flavor="context")
     ```
    to store the TikZ file as `mytikz.tex`.
 
-3. Add the contents of `mytikz.tex` into your LaTeX source code. A convenient way of
+3. Add the contents of `mytikz.tex` into your TeX source code. A convenient way of
    doing so is via
     ```latex
     \input{/path/to/mytikz.tex}
     ```
    Also make sure that the packages for PGFPlots and proper Unicode support and are
-   included in the header of your document:
-    ```latex
-    \usepackage[utf8]{inputenc}
-    \usepackage{fontspec}  % optional
-    \usepackage{pgfplots}
-    \pgfplotsset{compat=newest}
-    \usepgfplotslibrary{groupplots}
-    \usepgfplotslibrary{dateplot}
+   included in the header of your document. Get the code via:
+    ```python
+    import tikzplotlib
```

Comment:
Users will be too lazy to execute this command. Let's just keep the copy-pastable code in, and and add a code block for context, too, even if this means duplicating code from `Flavors`.

Reply:
Yeah, problem is that it already diverged, but sure, it’s your project :wink: